src/transform.js

```diff
@@ -33,6 +33,7 @@ export default class i18nTransform extends Transform {
       namespaceSeparator: ':',
       pluralSeparator: '_',
       output: 'locales/$LOCALE/$NAMESPACE.json',
+      defaultValueLocale: null,
```

Comment:
He did. That option implies a boolean, but if we don't know the source of truth locale, then either all translations will revert to defaults which won't work, or we would have to track possible changes in every locale, and at that point during the normal flow keys have been diverted to old, files have been written, etc. It's also not very practical to scan other all locales/namespaces to see if every key has changed, since then the merge function would need to know too much.

Reply:
> So, if this was set to e.g. en, that locale would get updated? Are there other ways to configure the "default locale" that this could conflict with?

All locales get reset if that locale has diverged from the default value. I couldn't find anything that lets you configure a default locale, which surprised me.  FWIW I suspect it's because the maintainers are French/Quebecois and they tend to be defensive about English being a default language.